app/Services/ArticleService.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace App\Services;
+
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Storage;
+
+class ArticleService
+{
+    private $filePath = null;
+    public function store(array $data){
+
+        try{
```

Comment:
@gemaantikahr I think you're doing it way overcomplicating things. Watch other pull requests, there's a really small chance that in this simple project, something would go wrong so that you would need transactions, repositories, try-catch with logging errors (I would actually prefer to use external service for that like Bugsnag), etc.